examples/rf.py

```diff
@@ -45,36 +49,19 @@ def rfr(cfg, seed):
         bootstrap=cfg["do_bootstrapping"],
         random_state=seed)
 
-    rmses = []
-    for train, test in kf:
-        # We iterate over cv-folds
-        X_train, X_test = X[train], X[test]
-        y_train, y_test = y[train], y[test]
-
-        rfr.fit(X_train, y_train)
-
-        y_pred = rfr.predict(X_test)
-
-        # We use root mean square error as performance measure
-        rmse = np.sqrt(np.mean((y_pred - y_test)**2))
-        rmses.append(rmse)
-    return np.mean(rmses)
+    def rmse(y, y_pred):
+        return np.sqrt(np.mean((y_pred - y)**2))
+    # Creating root mean square error for sklearns crossvalidation
+    rmse_scorer = make_scorer(rmse, greater_is_better=False)
+    score = cross_val_score(rfr, boston.data, boston.target, cv=11, scoring=rmse_scorer)
```

Comment:
Could you please rename the target function to not have the same name as the estimator object?